bugbug/models/spambug.py

```diff
@@ -87,5 +87,14 @@ def get_labels(self):
 
         return classes, [0, 1]
 
+    def items_gen(self, classes):
```

Comment:
Please add a comment to say that we overwrite the method so that we can include invalid bugs.